ui/src/App.vue

```diff
@@ -1,5 +1,10 @@
 <template>
   <div id="app">
+    <!-- <div>
+      <button v-for="entry in languages" :key="entry.title" @click="changeLocale(entry.language)">
+        <flag :iso="entry.flag" v-bind:squared="false" /> {{ entry.title }}
+      </button>
+    </div> -->
```

Comment:
I don't think you want any of this here. I think you want to use it in `ui/src/components/AppHeader/AppHeader.vue`
You can see here there is place to add things to the header bar. The user avatar is already there.

https://github.com/Call-for-Code-for-Racial-Justice/Five-Fifths-Voter/blob/e196f0c4fee5c50ef55f358330c777106ab2c675/ui/src/components/AppHeader/AppHeader.vue#L24-L28

https://vue.carbondesignsystem.com/?path=/story/components-cvuishell--header-base-with-navigation-and-actions

You would also need to set a value for each flag in the data component. 
```javascript
{ flag: 'us', language: 'en', title: 'English' },
```

Reply:
Thank you @davidnixon. It's commented out right now. I will move it to the user avatar section in a different PR.